website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md

```diff
@@ -79,3 +79,11 @@ https://console.aws.amazon.com/codesuite/codecommit/repositories/<repo>/pull-req
 ```
 https://dev.azure.com/<org>/<repo>/_git/dbt/pullrequestcreate?sourceRef={{source}}&targetRef={{destination}}
 ```
+
+### Configuring Custom Branches on Repositories
```

Comment:
If you make this `##` then the table of contents is going to look more sensible. Right now, this is rendering under the `Example templates` section.

I might actually recommend moving it up above the `Example templates` section with an `h2` altogether?